### PR TITLE
fix: numbers and emails not being populated

### PR DESCRIPTION
**Pull request by lwhiteley**
related to issue #45

### CODE REVIEW

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 1]
```diff
@@ -37,9 +43,15 @@ public void getAll(Callback callback) {
     ContentResolver cr = getReactApplicationContext().getContentResolver();
     Uri uri = ContactsContract.Contacts.CONTENT_URI;
     String selection = ContactsContract.Contacts.IN_VISIBLE_GROUP + " = " + 1;
+    Resources resources = this.reactContext.getResources();
     String sortBy = CommonDataKinds.Contactables.LOOKUP_KEY;
-
-    Cursor cursor = cr.query(uri, null, selection, null, sortBy);
+    String [] projection = new String[]{
+            CommonDataKinds.Contactables._ID,
+            CommonDataKinds.Contactables.DISPLAY_NAME,
```
Comment (morenoh149) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: can you replace this with family_name and given_name instead? It another issue we might as well solve here

Comment (lwhiteley) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: kool ill test it out

Comment (morenoh149) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: see https://github.com/rt2zz/react-native-contacts#example-contact-record

Comment (lwhiteley) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: turns out i cant just add them, it would require a different query n some more thought.. may do it in another changeset

**Comment by morenoh149** [comment]
and just to clarify module version 0.2.2 doesn't work for you as well? that was before this change https://github.com/rt2zz/react-native-contacts/commit/4e9c0f9cd8516e657cc3861110ca305e48ef212e

**Comment by lwhiteley** [comment]
nah it doesnt

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java (hunk 1 above)
Comment (npomfret) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: I think I've got a fix - https://github.com/rt2zz/react-native-contacts/pull/48

Have tested but only on older versions of android (and am going on holiday very soon so might not have time to test properly for a week or so).

Anyway - looks like it gets the data back fine.  I'll at least try and do some performance testing tomorrow.

Comment (lwhiteley) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: thought it was  a solution for the name thing but this is basically the same solution i have with multiple cursors. it should work on newer versions as well
